src/HttSystematics_MSSMvsSMRun2.cc

```diff
@@ -1764,44 +2101,59 @@ void AddMSSMvsSMRun2Systematics(CombineHarvester &cb, bool jetfakes, bool embedd
             for (auto m: masses) {
             double bbH_btag_up = js[s]["mssm_bbH_signals"][m]["btag_catagories"][c][yr]["Up"].asDouble();
             double bbH_btag_down = js[s]["mssm_bbH_signals"][m]["btag_catagories"][c][yr]["Down"].asDouble();
+            double ggH_btag_up = js[s]["mssm_ggH_signals"][m]["btag_catagories"][c][yr]["Up"].asDouble();
+            double ggH_btag_down = js[s]["mssm_ggH_signals"][m]["btag_catagories"][c][yr]["Down"].asDouble();
+            double bbH_nobtag_up = js[s]["mssm_bbH_signals"][m]["nobtag_catagories"][c][yr]["Up"].asDouble();
+            double bbH_nobtag_down = js[s]["mssm_bbH_signals"][m]["nobtag_catagories"][c][yr]["Down"].asDouble();
+            double ggH_nobtag_up = js[s]["mssm_ggH_signals"][m]["nobtag_catagories"][c][yr]["Up"].asDouble();
+            double ggH_nobtag_down = js[s]["mssm_ggH_signals"][m]["nobtag_catagories"][c][yr]["Down"].asDouble();
+            double bbH_ttbar_cr_up = js[s]["mssm_bbH_signals"][m]["ttbar_cr"][c][yr]["Up"].asDouble();
+            double bbH_ttbar_cr_down = js[s]["mssm_bbH_signals"][m]["ttbar_cr"][c][yr]["Down"].asDouble();
+            double ggH_ttbar_cr_up = js[s]["mssm_ggH_signals"][m]["ttbar_cr"][c][yr]["Up"].asDouble();
+            double ggH_ttbar_cr_down = js[s]["mssm_ggH_signals"][m]["ttbar_cr"][c][yr]["Down"].asDouble();
+
+            // tt channel numbers are swapped for btag and nobtag categories so swap round here
+            // if this are fixed in the json file need to swap back!!
```

Comment:
Wouldn't it be better to correct directly the .json file to avoid this here?

Reply:
Yes but the json file needs be reproduced, so in the meantime we can use the fix (George can remake the json when he is back)

Reply:
Ok, fine with me for the time-being, but this should not remain a permanent solution.